tests/unittests/test_log_filtering_functions.py

```diff
@@ -2,8 +2,8 @@
 # Licensed under the MIT License.
 import typing
 
-from azure_functions_worker import testutils
-from azure_functions_worker.testutils import TESTS_ROOT, remove_path
+from tests.utils import testutils
+from tests.utils.testutils import TESTS_ROOT, remove_path
```

Comment:
Similar comment here. We should simplify the testutils import.

Reply:
Fixed